src/modules/hub.bicep

```
@@ -0,0 +1,75 @@
+/**
+ * Parameters
+ */
+
+@description('Name of the hub. Used to ensure unique resource names.')
+param hubName string
+
+// Generate unique storage account name
+var storageAccountSuffix = 'store'
+var storageAccountName = '${substring(replace(toLower(hubName), '-', ''), 0, 24 - length(storageAccountSuffix))}${storageAccountSuffix}'
+
+@description('Specifies the location for resources. See https://aka.ms/azureregions.')
```

Aligning to the spec:
```suggestion
@description('Optional. Azure location where all resources should be created. See https://aka.ms/azureregions. Default: (resource group location).')
```